Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -72,4 +79,10 @@ protected void TakeDamage(int damage)
             isAlive = false;
     }
 
+    public void RemoveHealthBars()
```

Comment:
Just a nitpick, but could be renamed to `HideHealthBar` (although I do insist on removing that "s" 😛).

Reply:
Good point!